modules/google_analytics_analyzer/config/install/views.view.analyze_google_analytics.yml

```diff
@@ -0,0 +1,822 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - google_analytics_reports
+    - google_analytics_analyzer
+id: analyze_google_analytics
+label: 'Analyze Google Analytics'
+module: views
+description: ''
+tag: ''
+base_table: google_analytics
+base_field: ''
+display:
+  default:
+    id: default
+    display_title: Default
+    display_plugin: default
```

Comment:
I know the full report page is a free-for-all but we should still set the right example and promote UI consistency, all we have to do is make this a table and it should look more like the tabular data on the Analyze main page :)

Reply:
Hi @jjroelofs 

Sadly is doesn't, as Views renders the table as a left-to-right list of table headers and values, whereas your styling renders it as as a two-column table. I've made the change so you can see which you prefer.